data-builder/parse_dinosaur_bone_crafting.py

```diff
@@ -29,24 +32,57 @@ def get_systems_from_page(soup):
 
         body = table.find('tbody')
 
-        system_name = headers[0].getText()
+        system_name = headers[0].getText().rstrip()
```

Comment:
These items were coming back with a newline character at the end, which was causing a mismatch later.